test/ui-test/testSuites/suite_status/tst_groupChat/test.feature

```diff
@@ -0,0 +1,37 @@
+Feature: Status Desktop Group Chat
+
+	As a user I want to use group chat functionality.
+
+  	Scenario: As an admin user I want to create a group chat with my contacts and invited users can send messages
+
+  	   Given the user starts the application with a specific data folder ../../../testSuites/suite_status/tst_groupChat/data
+  	    When the user tester123 logs in with password TesTEr16843/!@00
+        Then the user lands on the signed in app
+        When the user creates a group chat adding users
+        	 | Athletic |
+        	 | Nervous  |
+		Then the group chat is created
```

Comment:
Yeah.. It could be.. Here we can open another pending discussion... do we separate the validations to have a more explicit syntax and then these steps can be more useful in other contexts, for example, the chat title validation, it can also be used to validate a community chat title channel, or do we encapsulate all of this in a "process" (it was my first idea, but then there was a discussion about having a more explicit syntax)?
What are your opinions on that @alaibe @jrainville @iurimatias ?

Reply:
I like having a step called "is created" or "appears" or whatever that validates. Like you said, it makes it so we can reuse it in other contexts. But no hard feelings if it's done the other way around.